Backoffice/Manager/BlockManager.php

```diff
@@ -67,6 +67,22 @@ public function initializeBlock($component, $siteId, $language, $isTransverse)
 
     /**
      * @param BlockInterface $block
+     * @param string         $language
+     *
+     * @return BlockInterface
+     */
+    public function createToTranslateBlock(BlockInterface $block, $language)
+    {
+        $block = clone $block;
+        $oldLanguage = $block->getLanguage();
+        $block->setLanguage($language);
+        $block->setLabel($block->getLabel()."[".$oldLanguage."]");
+
```

Comment:
@alavieille : is ``$block->getLabel()."[".$oldLanguage."]"`` still usefull after this ?